### PR TITLE
♻️ Tailwind

### PR DESCRIPTION
**Pull request by dankolbman**
Redo with create react app and tailwind.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,7 +1,3 @@
 {
-  "extends": ["airbnb", "prettier"],
```
Comment (bdolly) [inline .eslintrc.json]: keep prettier

File: package.json [hunk 2]
```diff
@@ -1,66 +1,34 @@
...
-  "scripts": {
-    "prepare": "rm -rf dist && mkdir dist && babel src --out-dir dist",
-    "storybook": "start-storybook -p 9001 -c .storybook",
-    "storybook:build": "build-storybook -c .storybook -o ./storybook-static/",
-    "storybook:build:serve": "build-storybook -c .storybook -o ./storybook-static/$npm_package_version && node versionBump.js && cd ./storybook-static && python -m SimpleHTTPServer"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/kids-first/kf-uikit.git"
+  "name": "app",
```
Comment (bdolly) [inline package.json]: add the meta fields that were here before

File: package.json [hunk 3]
```diff
@@ -1,66 +1,34 @@
...
-    "@emotion/babel-plugin-core": "^0.6.6",
-    "@storybook/addon-a11y": "^3.4.8",
-    "@storybook/addon-actions": "^3.4.8",
-    "@storybook/addons": "^3.4.8",
-    "babel-eslint": "^8.2.6",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-react-docgen": "^2.0.0-rc.1",
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb": "^17.0.0",
-    "eslint-config-prettier": "^2.9.0",
```
Comment (bdolly) [inline package.json]: re add this

File: package.json [hunk 4]
```diff
@@ -1,66 +1,34 @@
...
+    "@storybook/addon-knobs": "^3.4.10",
+    "eslint": "^5.3.0",
+    "eslint-config-airbnb": "^17.1.0",
     "eslint-loader": "^2.1.0",
-    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-prettier": "^2.6.2",
-    "eslint-plugin-react": "^7.10.0",
-    "prettier": "^1.14.0",
```
Comment (bdolly) [inline package.json]: re-add this

File: package.json [hunk 5]
```diff
@@ -1,66 +1,34 @@
...
-    "react-docgen": "^2.21.0",
```
Comment (bdolly) [inline package.json]: re-add this

File: package.json [hunk 6]
```diff
@@ -1,66 +1,34 @@
...
-    "paths.macro": "^2.0.2",
-    "@storybook/react": "^3.4.8",
-    "@storybook/addon-info": "^3.4.8",
-    "@storybook/addon-knobs": "^3.4.8",
-    "chroma-js": "^1.3.7",
-    "create-emotion": "^9.2.6",
-    "create-emotion-styled": "^9.2.6",
-    "emotion": "^9.2.5",
-    "emotion-theming": "^9.2.5",
-    "lodash": "^4.17.10",
```
Comment (bdolly) [inline package.json]: re-add this

File: src/components/buttons/Button.jsx [hunk 7]
```diff
@@ -1,59 +0,0 @@
-import React from 'react';
```
Comment (bdolly) [inline src/components/buttons/Button.jsx]: keep this file

File: src/theme/Buttons.js [hunk 8]
```diff
@@ -1,44 +0,0 @@
-import chroma from 'chroma-js';
```
Comment (bdolly) [inline src/theme/Buttons.js]: I think we can keep the thene, just need to figure out how to roll it into tailwind

File: src/theme/Colors.js [hunk 9]
```diff
@@ -1,71 +0,0 @@
-import chroma from 'chroma-js';
```
Comment (bdolly) [inline src/theme/Colors.js]: maybe keep this and re-use in tailwind configs ?

File: src/theme/Typography.js [hunk 10]
```diff
@@ -1,167 +0,0 @@
-import { zipObject } from 'lodash';
```
Comment (bdolly) [inline src/theme/Typography.js]: ☝️ see if we can reuse this in tailwind

File: src/utils/colors.js [hunk 11]
```diff
@@ -1,52 +0,0 @@
-import chroma from 'chroma-js';
```
Comment (bdolly) [inline src/utils/colors.js]: keep this util file

File: stories/Colors/Colors.jsx [hunk 12]
```diff
@@ -1,114 +1,30 @@
...
-import styled from '../../src/kfFeels/kfReactEmotion';
-import { H3 } from '../../src/components/Typography';
-import { Paragraph } from '../../src/components/Typography/Text';
-import { Box } from '../../src/components/Layout';
-import { wcagGrade } from '../../src/utils/colors';
 
 /**
  * A basic square swatch to display a color
  */
-/* eslint-disable no-shadow */
```
Comment (bdolly) [inline stories/Colors/Colors.jsx]: I think we can keep the storybook specific components but just style them with tailwind

File: stories/Colors/Colors.story.jsx [hunk 13]
```diff
@@ -1,136 +1,34 @@
...
-      display: 'flex',
-      flexWrap: 'wrap',
-      flexDirection: 'row',
-    }}
-  >
-    {Object.keys(brandColors).map(name => (
-      <ColorSpecimen key={name} color={brandColors[name].hex} name={name} />
-    ))}
+const brand = ['primary', 'secondary', 'tertiary'];
+const brandSwatches = brand.map(color => <Swatch color={color} />);
```
Comment (bdolly) [inline stories/Colors/Colors.story.jsx]: this is where we can leverage the original js theme for both tailwind configs and storbook. The theme will be the single point of truth

File: versionBump.js [hunk 14]
```diff
@@ -1,28 +0,0 @@
-const replace = require('replace-in-file');
-const pjVersion = require('./package.json').version;
-const fs = require('fs');
-
-const options = [
-  {
-    files: 'storybook-config.json',
```
Comment (bdolly) [inline versionBump.js]: will need to figure out a way to update the storybook-config on build and deploy so that the versions addon works

Comment (dankolbman) [reply versionBump.js]: I think this is more of a job for the ci/cd flow.

File: src/components/buttons/Button.jsx (hunk 7 above)
Comment (dankolbman) [reply src/components/buttons/Button.jsx]: Will re-write this for tailwinds

File: src/theme/Buttons.js (hunk 8 above)
Comment (dankolbman) [reply src/theme/Buttons.js]: wouldn't a different tailwinds.js file be equivalent?

File: stories/Colors/Colors.jsx (hunk 12 above)
Comment (dankolbman) [reply stories/Colors/Colors.jsx]: Need to re-write them cause we no longer have emotion.
Also not sure how to pull out hex values.

**Pull request merged**